README.md

```diff
@@ -184,21 +184,27 @@ This will produce a significantly smaller download - in the order of 22-30 MB; w
    b. *OR* configure and compile VS manually, using the ncurses ccmake frontend:
 
    ```bash
-   mkdir build & cd build
+   mkdir build
+   cd build
```

Comment:
Why this change? What was wrong with having both commands on the same line, the way it was before?

Reply:
The problem was the wrong `&` instead of the intended `&&`
I don't understand the need to chain the commands with `&&` for that line but not the following ones.
I went for the simple solution of separating the commands

Reply:
OK, I guess that makes sense.

Reply:
yeah it's fine

@vincele said:
> I don't understand the need to chain the commands with && for that line but not the following ones.

It basically moves it into the directory for that line and then the following lines just work in that directory.
It's fine to separate it out or use `pushd` to change into the directory instead.